packages/extension/src/view/devtools/components/webStories/contentPanel.tsx

```diff
@@ -17,24 +17,117 @@
  * External dependencies.
  */
 import React from 'react';
+import { ChipsBar, FiltersSidebar, TopBar } from '@google-psat/design-system';
+import { Resizable } from 're-resizable';
+import { noop } from '@google-psat/common';
+
 /**
  * Internal dependencies.
  */
 import { getStoryMarkup } from './createStoryIframe';
 import { STORY_JSON } from './story';
+import { useStories } from '../../stateProviders';
+
+interface WebStoriesProps {
+  storyOpened: boolean;
+}
 
-const WebStories = () => {
+const WebStories = ({ storyOpened }: WebStoriesProps) => {
   const storyMarkup = getStoryMarkup(STORY_JSON);
 
+  const {
+    searchValue,
+    setSearchValue,
+    showFilterSidebar,
+    setShowFilterSidebar,
+    setSortValue,
+    sortValue,
+    selectedFilters,
+    toggleFilterSelection,
+    resetFilters,
+    selectedFilterValues,
+    filters,
+  } = useStories(({ state, actions }) => ({
+    searchValue: state.searchValue,
+    filters: state.filters,
+    sortValue: state.sortValue,
+    selectedFilterValues: state.selectedFilterValues,
+    setSearchValue: actions.setSearchValue,
+    showFilterSidebar: state.showFilterSidebar,
+    setShowFilterSidebar: actions.setShowFilterSidebar,
+    setSortValue: actions.setSortValue,
+    selectedFilters: state.selectedFilters,
+    toggleFilterSelection: actions.toggleFilterSelection,
+    resetFilters: actions.resetFilters,
+  }));
+
   return (
-    <div
-      data-testid="web-stories-content"
-      className="h-full w-full text-raisin-black dark:text-bright-gray overflow-y-auto"
-    >
-      <iframe
-        srcDoc={storyMarkup}
-        style={{ width: '100%', height: '100vh', border: 'none' }}
-      />
+    <div className="h-full w-full flex flex-col">
+      {!storyOpened && (
+        <>
+          <TopBar
+            searchValue={searchValue}
+            setSearchValue={setSearchValue}
+            showFilterSidebar={showFilterSidebar}
+            setShowFilterSidebar={setShowFilterSidebar}
+            hideFiltering={false}
+            disableFiltering={false}
+            hideSearch={false}
+            count={0} // TODO: Add count
+          >
+            <div className="flex justify-between min-w-28">
+              <p>Sort by:</p>
```

Comment:
Should this be `label`?

Reply:
Nope, key is the object access key.
Eg. analytics.category in Cookies table